features/networking/egress-ingress.feature

```diff
@@ -662,3 +662,47 @@ Feature: Egress-ingress related networking scenarios
     When I execute on the pod:
       | curl | --connect-timeout | 5 | --head | www.chsi.com.cn |
     Then the step should fail
+
+
+  # @author jechen@redhat.com
+  # @case_id OCP-44940
+  @admin
+  @4.10 @4.9    
+  Scenario: [bug2000057] No segmentation error occurs in ovnkube-master after egressfirewall resource that references a DNS name is deleted
+    Given the env is using "OVNKubernetes" networkType
+    And I have a project
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+		
+    When I obtain test data file "networking/ovn-egressfirewall/egressfirewall-policy4.yaml"
+    And I run oc create as admin over "egressfirewall-policy4.yaml" replacing paths:
+      | ["metadata"]["namespace"]  | <%= project.name %>   |
+    Then the step should succeed
```

Comment:
Suggest add one step to make sure the egressfirewall was applied successfully.  
oc get egressfirewall -n $namespace  , should have below output.
EgressFirewall Rules applied